backend/src/services/project/project-service.ts

```diff
@@ -681,11 +688,19 @@ export const projectServiceFactory = ({
     actor
   }: TListProjectCasDTO) => {
     const project = await projectDAL.findProjectByFilter(filter);
+    let projectId = project.id;
+    const certManagerProjectFromSplit = await projectDAL.getProjectFromSplitId(
```

Comment:
is it possible to do this more in some kind of middleware? feels like a lot of repeated code